devicon.json

```diff
@@ -7335,6 +7335,37 @@
             }
         ]
     },
+    {
+        "name": "readthedocs",
+        "altnames": [
+            "read the docs",
```

Comment:
Maybe we can keep only "Read the Docs" here. I think we can use lowercase in search engine if it's not already. 🤔

Reply:
@ConX For quick application if you agree with lunatic :)
```suggestion
```

Reply:
I agree with this. 👍